skimage/util/_iterator_tests.pyx

```diff
@@ -0,0 +1,17 @@
+include "npy_iterator.pxi"
+
+
+def run_iterator_tests():
+    arr = np.arange(720).reshape(6, 5, 4, 3, 2).astype(np.int)
+
+    cdef Iterator iter = Iterator(arr)
```

Comment:
(This is probably related to my question about importing numpy...) I don't know how pure-C imports work in Cython, but isn't there a way to make them "scoped" by namespace as in Python?